README.md

```diff
@@ -1,4 +1,5 @@
-# Antrea
+![Antrea Logo](/docs/assets/antrea_logo.svg)
```

Comment:
add the logo after the status?

Reply:
I looked at other vmware-tanzu projects, they are doing the same thing. I prefer it this way, but I guess it's a subjective matter...